gsap/Animation.d.ts

```diff
@@ -0,0 +1,90 @@
+        play(from?: any, suppressEvents?: boolean): Animation;
+
+        /** Gets or sets the animations's progress which is a value between 0 and 1 indicating the position of the virtual playhead (excluding repeats) where 0 is at the beginning, 0.5 is at the halfway point, and 1 is at the end (complete). */
+        progress(): number;
+        progress(value: number, suppressEvents?: boolean): Animation;
+
+        /** Restarts and begins playing forward from the beginning. */
+        restart(includeDelay?: boolean, suppressEvents?: boolean): Animation;
+
+        /** Resumes playing without altering direction (forward or reversed), optionally jumping to a specific time first. */
+        resume(from?: any, suppressEvents?: boolean): Animation;
+
+        /** Reverses playback so that all aspects of the animation are oriented backwards including, for example, a tween's ease. */
+        reverse(from?: any, suppressEvents?: boolean): Animation;
+
+        /** Gets or sets the animation's reversed state which indicates whether or not the animation should be played backwards. */
+        reversed(): boolean;
+        reversed(value: boolean): Animation;
+
+        /** Jumps to a specific time without affecting whether or not the instance is paused or reversed. */
+        seek(time: any, suppressEvents?: boolean): Animation;
+
+        /** Gets or sets the time at which the animation begins on its parent timeline (after any delay that was defined). */
+        startTime(): number;
+        startTime(value: number): Animation;
+
+        /** Gets or sets the local position of the playhead (essentially the current time), described in seconds (or frames for frames-based animations) which will never be less than 0 or greater than the animation's duration. */
+        time(): number;
+        time(value: number, suppressEvents?: boolean): Animation;
+
+        /** Factor that's used to scale time in the animation where 1 = normal speed (the default), 0.5 = half speed, 2 = double speed, etc. */
+        timeScale(): number;
+        timeScale(value: number): Animation;
+
+        /** Gets or sets the animation's total duration including any repeats or repeatDelays (which are only available in TweenMax and TimelineMax). */
+        totalDuration(): number;
+        totalDuration(value: number): Animation;
+
+        /** Gets or sets the animation's total progress which is a value between 0 and 1 indicating the position of the virtual playhead (including repeats) where 0 is at the beginning, 0.5 is at the halfway point, and 1 is at the end (complete). */
+        totalProgress(): number;
+        totalProgress(value: number, suppressEvents?: boolean): Animation;
+
+        /** Gets or sets the position of the playhead according to the totalDuration which includes any repeats and repeatDelays (only available in TweenMax and TimelineMax). */
+        totalTime(): number;
+        totalTime(time: number, suppressEvents?: boolean): Animation;
```

Comment:
@DanielRosenwasser Unless i'm misunderstanding that alters the return type into a manner that i find un-intuitive.
`totalTime(time?: number, suppressEvents?: boolean): Animation | boolean;` 

Just looks confusing. Is this what you meant?